apps/dev-portal/lib/gql/miniapp.graphql

```diff
@@ -0,0 +1,73 @@
+query allApps($creator: UserWhereInput!, $first: Int!) {
+    b2c: allB2CApps(sortBy: createdAt_DESC, where: { createdBy: $creator }, first: $first) {
+        id
+        name
```

Comment:
Just a question, is this autogen or not?

Reply:
This one is not. `.ts` one is